executor/aggfuncs/aggfunc_test.go

```diff
@@ -234,6 +234,124 @@ func distinctUpdateMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType)
 	return memDeltas, nil
 }
 
+func maxUpdateMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType) (memDeltas []int64, err error) {
+	memDeltas = make([]int64, srcChk.NumRows())
+	var (
+		preStringVal string
+		preJSONVal   string
+		preEnumVal   types.Enum
+		preSetVal    types.Set
```

Comment:
IMO, we don't need those vars, just need a `preMemUsage int64`.

Reply:
Those variables are used to compare values, to find the bigger value. Is this right?

Reply:
ok, you are right.

Reply:
the conflicting file `executor/aggfuncs/aggfunc_test.go` need to be resolved.